common/src/main/java/com/wynntils/features/utilities/TranslationFeature.java

```diff
@@ -120,12 +120,17 @@ public void onNpcDialogue(NpcDialogueProcessingEvent.Pre event) {
     }
 
     private StyledText unwrapCoding(String origCoded) {
-        // Some translated text (e.g. from pt_br) contains Á. This will be stripped later on,
-        // so convert it to A (not ideal but better than nothing).
-        // FIXME: Check if Á should be À
+        // Some translated text (e.g. from pt_br) contains special characters.
+        // These will need to be stripped or converted, which is not ideal but better than nothing.
```

Comment:
Just thinking out loud, is there some way we can preserve these, by encoding them differently? E.g. replace it with `§&Aacute;` and then search for that string and replace it back with `Á` after all stripping is done?

Or make the stripping before the translation, so we don't need to strip after translating, when  `Á` might be a valid character.

Or, if we cannot strip before translation, do it the opposite way, as soon as we can, replace  `Á` with `§_` or something like that, and then strip these markers?

Reply:
These characters cannot be put in chat either way, so we need to strip after translation, since they are changed in the Wynn resource pack, and render weirdly. I also don't think we can use Unicode codes for these characters, as essentially, they will render the same font.

Reply:
Ah, I see. If we ever are going to fix that, we'd need to combine it with a patch to the Wynn resource pack. I'll leave it to the Portuguese speaking users to file a bug if they deem it important to fix.